azure-iot-device/tests/common/pipeline/test_pipeline_stages_mqtt.py

```diff
@@ -298,36 +261,40 @@ def provider_function_throws_base_exception(params, stage, mocker, fake_base_exc
 
 @pytest.mark.parametrize("params", connection_ops + pubsub_ops)
 @pytest.mark.describe(
-    "MQTT Provider pipeline stage _run_op with ops that map directly to provider libraray calls"
+    "Provider - .run_op() -- called with op that maps directly to protocol client library calls"
 )
 class TestMQTTProviderBasicFunctionality(object):
-    @pytest.mark.it("calls the appropriate function on the provider library")
+    @pytest.mark.it("Calls the appropriate function on the protocol client library")
     def test_calls_provider_function(self, stage, create_provider, params, op):
         stage.run_op(op)
         assert getattr(stage.provider, params["provider_function"]).call_count == 1
 
-    @pytest.mark.it("passes the correct args to the provider function")
+    @pytest.mark.it("Passes the correct args to the protocol client library function")
     def test_passes_correct_args_to_provider_function(self, stage, create_provider, params, op):
```

Comment:
Not sure I think this should be a separate test. I think passing the correct args is implied by testing the calling, right? I'd personally combine this test with the one above.